src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import Content from "./components/Content.jsx";
+import Header from "./components/Header.jsx";
+import { createElement } from "./createElement.js";
```

Comment:
```suggestion
// 자동 import 설정
```
매번 `createElement` 함수를 import 하지 않아도 자동으로 import 해주는 기능이 있는 것 같아서 한번 찾아보셔도 좋을 것 같아요~